libraries/SITL/SITL.h

```diff
@@ -98,6 +98,8 @@ class SITL
     AP_Int16  mag_delay; // magnetometer data delay in ms
     AP_Int16  wind_delay; // windspeed data delay in ms
 
+    AP_Int16 gps_spd_err; // reported GPS speed accuracy in cm/s
```

Comment:
a bit of a confusing variable name. maybe it should gps_spd_acc so people don't think it's a controller error value?